scripts/scr_poststage.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# SCR allows you to spawn off dataset transfers "in the background"
+# that will finish some time after a job completes.  This saves you from
+# using your compute time to transfer datasets.  You can do this by
+# specifying XFER=BBAPI_POSTSTAGE for a storage descriptor in scr.conf.
+# Currently, this is only supported on on IBM burst buffer nodes.
+#
+# This script is to be run as a 2nd-half post-stage script on an IBM
+# system.  A 2nd-half post-stage script will run after all the job's burst
+# buffers transfers have finished (which could be hours later after the job
+# finishes).  This script will finalize any completed burst buffer dataset
+# transfers so that they're visible to SCR.
+#
+
+# Please fill in BINDIR AND PREFIX below:
+
+# The full path to the dir containing the SCR binaries (scr_index and scr_flush_file)
+BINDIR=""
```

Comment:
I think we could prefill this value for people by changing this to a ``.in`` file and processing it during the cmake configure step.
```
BINDIR="@X_BINDIR@"
```
We do this for other scripts like ``scr_postrun.in``:
https://github.com/LLNL/scr/blob/55c800d60d621d538c48d1db2cc692a07477273a/scripts/common/scr_postrun.in#L24